### PR TITLE
German translation for settings page

### PR DESCRIPTION
**Pull request by SourceDoctor**
german language support for new settings area

### CODE REVIEW

File: resources/lang/de/settings.php [hunk 1]
```diff
@@ -0,0 +1,814 @@
...
+            'description' => 'Prüfe auf doppelte IPs wenn ein Gerät hinzugefügt wird',
+            'help' => 'Wenn ein Gerät als IP Adresse hinzugefügt wird, wird geprüft das diese IP Adresse noch nicht vorhanden ist. Ist die IP Adresse schon vorhanden wird sie nicht hinzugefügt. Wird das Gerät als Hostname hinzugefügt wird dieser Check nicht durchgeführt. Ist diese Einstellung aktiviert, werden die Hostnamen aufgelöst und der Check wird ebenso durchgeführt. Dies vermeidet versehentliches hinzufügen von Geräteduplikaten.'
+        ],
+        'alert' => [
+            'ack_until_clear' => [
+                'description' => 'Standard Meldung bis der Alarm abklingt',
+                'help' => 'Standard Meldung bis der Alarm abklingt'
+            ],
+            'admins' => [
+                'description' => 'Melde Alarme den Administratoren',
```
Comment (f0o) [inline resources/lang/de/settings.php]: Replace `Melde` with `Benachrichtige`

File: resources/lang/de/settings.php [hunk 2]
```diff
@@ -0,0 +1,814 @@
...
+                'help' => 'Alarmiere Administratoren'
+            ],
+            'default_copy' => [
+                'description' => 'Sende alle E-Mail Alarme als Kopie zum Standard Kontakt',
```
Comment (f0o) [inline resources/lang/de/settings.php]: Make use of `Benachrichtige`, it's a wonderful verb for these things

File: resources/lang/de/settings.php [hunk 3]
```diff
@@ -0,0 +1,814 @@
...
+                'help' => 'Sende alle E-Mail Alarme als Kopie zum Standard Kontakt'
```
Comment (f0o) [inline resources/lang/de/settings.php]: ☝️

File: resources/lang/de/settings.php [hunk 4]
```diff
@@ -0,0 +1,814 @@
...
+            ],
+            'default_if_none' => [
+                'description' => 'Kann nicht auf der Webseite konfiguriert werden?',
+                'help' => 'Sende E-Mail zum Standard Kontakt wenn keine anderen Kontakte gefunden werden'
```
Comment (f0o) [inline resources/lang/de/settings.php]: ☝️

File: resources/lang/de/settings.php [hunk 5]
```diff
@@ -0,0 +1,814 @@
...
+            ],
+            'default_mail' => [
+                'description' => 'Standard Kontakt',
+                'help' => 'Der Standard E-Mail Kontakt'
```
Comment (f0o) [inline resources/lang/de/settings.php]: ☝️

File: resources/lang/de/settings.php [hunk 6]
```diff
@@ -0,0 +1,814 @@
...
+            ],
+            'default_only' => [
+                'description' => 'Sende Alarme nur zum Standard Kontakt',
```
Comment (f0o) [inline resources/lang/de/settings.php]: ☝️

File: resources/lang/de/settings.php [hunk 7]
```diff
@@ -0,0 +1,814 @@
...
+                'help' => 'Alarmiere nur den Standard E-Mail Kontakt'
+            ],
+            'disable' => [
+                'description' => 'Deaktivere Alarmierungen',
```
Comment (f0o) [inline resources/lang/de/settings.php]: Grammar and `Alarmierungen` sounds very wrong

File: resources/lang/de/settings.php [hunk 8]
```diff
@@ -0,0 +1,814 @@
...
+                'help' => 'Stoppe Generierung von Alarmen'
```
Comment (f0o) [inline resources/lang/de/settings.php]: Makes little sense, should be something more useful

File: resources/lang/de/settings.php [hunk 9]
```diff
@@ -0,0 +1,814 @@
...
+            ],
+            'fixed-contacts' => [
+                'description' => 'Update der Kontakt E-Mail Adresse werden nicht berücksichtigt',
```
Comment (f0o) [inline resources/lang/de/settings.php]: E-Mail should be plural

File: resources/lang/de/settings.php [hunk 10]
```diff
@@ -0,0 +1,814 @@
...
+                'help' => 'Wenn aktiviert werden jegliche Änderungen in sysContact oder Benutzer E-Mails nicht berücksichtigt solange Alarme aktiv sind'
+            ],
+            'globals' => [
+                'description' => 'Melde Alarme zu Benutzer welche nur Leserechte haben',
```
Comment (f0o) [inline resources/lang/de/settings.php]: `s/Melde/Benachrichtige/`

File: resources/lang/de/settings.php [hunk 11]
```diff
@@ -0,0 +1,814 @@
...
+                'help' => 'Alarmiere Administratoren welche nur Leserechte haben'
```
Comment (f0o) [inline resources/lang/de/settings.php]: This sentence is wrong - this is not regarding admins but RO-Users

File: resources/lang/de/settings.php [hunk 12]
```diff
@@ -0,0 +1,814 @@
...
+            ],
+            'syscontact' => [
+                'description' => 'Melde Alarme an sysContact',
```
Comment (f0o) [inline resources/lang/de/settings.php]: `s/Melde/Benachrichtige/`

File: resources/lang/de/settings.php [hunk 13]
```diff
@@ -0,0 +1,814 @@
...
+                'help' => 'Sende AlarmierungsE-Mail an den SNMP sysContact'
```
Comment (f0o) [inline resources/lang/de/settings.php]: `s/Melde/Benachrichtige/` & Grammar

File: resources/lang/de/settings.php [hunk 14]
```diff
@@ -0,0 +1,814 @@
...
+            ],
+            'transports' => [
+                'mail' => [
+                    'description' => 'Aktiviere Alarmierungen via E-Mail',
```
Comment (f0o) [inline resources/lang/de/settings.php]: `s/Alarmierungen/Benachrichtigungen/`

File: resources/lang/de/settings.php [hunk 15]
```diff
@@ -0,0 +1,814 @@
...
+                    'help' => 'Benachrichtige über Alarme via E-Mail'
```
Comment (f0o) [inline resources/lang/de/settings.php]: Not sure it needs to be stated that this is for alarms, that should be clear from the context

File: resources/lang/de/settings.php [hunk 16]
```diff
@@ -0,0 +1,814 @@
...
+                'help' => 'Toleranzfenster in Sekunden'
+            ],
+            'users' => [
+                'description' => 'Benachrichtige normale Benutzer über Alarme',
+                'help' => 'Alarmiere normale Benutzer'
+            ]
+        ],
+        'alert_log_purge' => [
+            'description' => 'Entferne Alarm Logeinträge älter als',
+            'help' => 'Bereinigung wirde erledigt von daily.sh',
```
Comment (f0o) [inline resources/lang/de/settings.php]: `s/wirde/wird` - Bereinigung sounds a bit wrong in this context but for lack of better alternative it should be fine

File: resources/lang/de/settings.php [hunk 17]
```diff
@@ -0,0 +1,814 @@
+<?php
+
+return [
+    'readonly' => 'Konfiguriere in config.php, entferne aus config.php zur Aktivierung.',
+    'groups' => [
+        'alerting' => 'Alarmierung',
+        'auth' => 'Authentifizierung',
```
[the remark this answers is not in the document]
Comment (SourceDoctor) [reply resources/lang/de/settings.php]: Authentication == Authentifizierung ;-)

**Comment by SourceDoctor** [comment]
@f0o first round of language fixes done ;)

File: resources/lang/de/settings.php [hunk 18]
```diff
@@ -0,0 +1,814 @@
...
+            'dashboard' => 'Dashboard Einstellungen',
+            'search' => 'Sucheinstellungen',
+            'style' => 'Theme',
+        ]
+    ],
+    'settings' => [
+        'active_directory' => [
+            'users_purge' => [
+                'description' => 'Behalte inaktive Nutzer für',
+                'help' => 'Benutzer werden von LibreNMS gelöscht nach diesen Tagen Inaktivität. 0 bedeutet nie und der Benutzer wird neu erstellt wenn er sich erneut anmeldet',
```
[the remark this answers is not in the document]
Comment (SourceDoctor) [reply resources/lang/de/settings.php]: changed

File: resources/lang/de/settings.php [hunk 19]
```diff
@@ -0,0 +1,814 @@
...
+        'auth' => [
+            'general' => 'Generelle Authentifizierungseinstellungen',
+            'ad' => 'Active Directory Einstellungen',
+            'ldap' => 'LDAP Einstellungen'
+        ],
+        'discovery' => [
+            'general' => 'Generelle Discovery Einstellungen'
+        ],
+        'external' => [
+            'binaries' => 'Binary Locations',
```
[the remark this answers is not in the document]
Comment (SourceDoctor) [reply resources/lang/de/settings.php]: changed

File: resources/lang/de/settings.php [hunk 20]
```diff
@@ -0,0 +1,814 @@
+<?php
+
+return [
+    'readonly' => 'Konfiguriere in config.php, entferne aus config.php zur Aktivierung.',
```
[the remark this answers is not in the document]
Comment (SourceDoctor) [reply resources/lang/de/settings.php]: changed

File: resources/lang/de/settings.php [hunk 21]
```diff
@@ -0,0 +1,814 @@
...
+    'groups' => [
+        'alerting' => 'Alarmierung',
```
[the remark this answers is not in the document]
Comment (SourceDoctor) [reply resources/lang/de/settings.php]: changed

File: resources/lang/de/settings.php (hunk 12 above)
Comment (SourceDoctor) [reply resources/lang/de/settings.php]: changed

File: resources/lang/de/settings.php (hunk 14 above)
Comment (SourceDoctor) [reply resources/lang/de/settings.php]: changed

File: resources/lang/de/settings.php (hunk 16 above)
Comment (SourceDoctor) [reply resources/lang/de/settings.php]: changed

**Comment by PipoCanaja** [comment]
@SourceDoctor @f0o  : Let me know if the PR is ready for merge :)

**Comment by SourceDoctor** [comment]
@PipoCanaja 
for me it's ready for merge. Don't know if @f0o  found further issues :-)

**Comment by PipoCanaja** [comment]
Let's go then :)

**Comment by justinh-rahb** [comment]
This commit breaks settings page.
Screen Shot 2019-11-05 at 10 34 19 PM

**Comment by PipoCanaja** [comment]
@justinh-rahb Will check and get this fixed

**Comment by SourceDoctor** [comment]
@PipoCanaja seems to be a Theme Issue.
in default Theme Translation works well.
responsible translation Library is `resources/lang/<lang>.json`
hope it helps :-)

**Comment by PipoCanaja** [comment]
Hi @justinh-rahb
You can run ./daily.sh to get this fixed. 
Thanx and bye.

**Comment by f0o** [comment]
So this got merged with all the typos and 20+ remarks I left?

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-58-release-changelog-november-2019/10224/1

**Pull request merged**